core/src/test/scala/unit/kafka/utils/JsonTest.scala

```diff
@@ -45,22 +49,64 @@ class JsonTest {
   }
 
   @Test
-  def testJsonEncoding() {
-    assertEquals("null", Json.encode(null))
-    assertEquals("1", Json.encode(1))
-    assertEquals("1", Json.encode(1L))
-    assertEquals("1", Json.encode(1.toByte))
-    assertEquals("1", Json.encode(1.toShort))
-    assertEquals("1.0", Json.encode(1.0))
-    assertEquals("\"str\"", Json.encode("str"))
-    assertEquals("true", Json.encode(true))
-    assertEquals("false", Json.encode(false))
-    assertEquals("[]", Json.encode(Seq()))
-    assertEquals("[1,2,3]", Json.encode(Seq(1,2,3)))
-    assertEquals("[1,\"2\",[3]]", Json.encode(Seq(1,"2",Seq(3))))
-    assertEquals("{}", Json.encode(Map()))
-    assertEquals("{\"a\":1,\"b\":2}", Json.encode(Map("a" -> 1, "b" -> 2)))
-    assertEquals("{\"a\":[1,2],\"c\":[3,4]}", Json.encode(Map("a" -> Seq(1,2), "c" -> Seq(3,4))))
+  def testLegacyEncodeAsString() {
+    assertEquals("null", Json.legacyEncodeAsString(null))
+    assertEquals("1", Json.legacyEncodeAsString(1))
+    assertEquals("1", Json.legacyEncodeAsString(1L))
+    assertEquals("1", Json.legacyEncodeAsString(1.toByte))
+    assertEquals("1", Json.legacyEncodeAsString(1.toShort))
+    assertEquals("1.0", Json.legacyEncodeAsString(1.0))
+    assertEquals("\"str\"", Json.legacyEncodeAsString("str"))
+    assertEquals("true", Json.legacyEncodeAsString(true))
+    assertEquals("false", Json.legacyEncodeAsString(false))
+    assertEquals("[]", Json.legacyEncodeAsString(Seq()))
+    assertEquals("[1,2,3]", Json.legacyEncodeAsString(Seq(1,2,3)))
+    assertEquals("[1,\"2\",[3]]", Json.legacyEncodeAsString(Seq(1,"2",Seq(3))))
+    assertEquals("{}", Json.legacyEncodeAsString(Map()))
+    assertEquals("{\"a\":1,\"b\":2}", Json.legacyEncodeAsString(Map("a" -> 1, "b" -> 2)))
+    assertEquals("{\"a\":[1,2],\"c\":[3,4]}", Json.legacyEncodeAsString(Map("a" -> Seq(1,2), "c" -> Seq(3,4))))
```

Comment:
Btw, we can use triple quotes to avoid the escaping here and in the other tests.

Reply:
Yep, done.